transcribe.py

```
@@ -534,34 +538,36 @@ def transcribe_project(name, model, language, only=None):
+            # genau die zu kurze Zahl, die den Verlust verdeckt. Der Rueckfall gilt Laeufen ohne
+            # das Feld (alte Roh-JSON, whisper.cpp ohne lesbare WAV).
+            dur = result.get("duration") or (result["segments"][-1]["end"] if result["segments"] else 0)
+            n_ok += 1
+            audio_gesamt += dur
+            print(f"[{name}] fertig {base}: {dt:.0f}s, {len(result['segments'])} Segmente, "
+                  f"Audio {fmt(dur)}, {dur/max(dt,1):.1f}x", flush=True)
+            if result["luecken"]:
+                orte = ", ".join(f"{fmt(x['start'])}-{fmt(x['end'])}" for x in result["luecken"])
+                print(f"[{name}] ⚠ {base}: {len(result['luecken'])} Abschnitt(e) ohne Transkript "
+                      f"({orte}) — bitte im Ton gegenhoeren", flush=True)
         except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)